tools/apigee-x-trial-provision/apigee-x-trial-provision.sh

```diff
@@ -315,11 +315,29 @@ gcloud compute forwarding-rules create apigee-proxy-https-lb-rule \
 
 set -e
 
-# TODO: [ ] wait till LB is ready
 echo ""
-echo "Almost done. It take some time (another 5-8 minutes) to provision load balancer infrastructure."
+echo "Almost done. It take some time (another 5-8 minutes) to provision the load balancer infrastructure."
 echo ""
 
+# TODO: more intelligent wait until LB is ready
+
+set +e
+while true
+do
+  SSL_STATUS="$(gcloud compute ssl-certificates list --format=json | jq -r '.[0].type')"
```

Comment:
i know we cant change the gcloud commands, but lets say 'TLS' not 'SSL' in our variables wherever possible. ref <https://bfy.tw/QgOl>

Reply:
ACK